config/routes.rb

```diff
@@ -26,7 +26,7 @@
 
   get '/cart', :to => 'carts#index', :as => 'cart'
 
-  resources :items, only: [:index, :show]
+  resources :items, only: [:show]
```

Comment:
Did we remember to delete the item/index.html?

Reply:
@Maxscores I actually just did. Thanks for the reminder